Add documentation for THM kernels and scalar kernels (#25077)
idaholab/moose: modules/thermal_hydraulics/doc/content/source/base/FlowModelSinglePhase.md

# FlowModelSinglePhase

> The `FlowModelSinglePhase` creates for [FlowChannel1Phase.md] components using the three equation model:

Author: ```suggestion
The `FlowModelSinglePhase` creates for [FlowChannel1Phase.md] components using the three equations model:
```

> The `FlowModelSinglePhase` creates for [FlowChannel1Phase.md] components using the three equations model:

Reviewer: ```suggestion
The `FlowModelSinglePhase` creates the following for [FlowChannel1Phase.md] components using the 3-equation model:
```

- variables
- initial conditions

> - kernels and DGKernels for RDG

Reviewer: ```suggestion
- kernels and DGKernels
```
